src/store/DevfileRegistries/index.ts

```diff
@@ -171,7 +171,7 @@ export const actionCreators: ActionCreators = {
         const patchedJSONString = JSON.stringify(schemav1).replaceAll('#/definitions', '#/oneOf/0/definitions');
         const parsedSchemaV1 = JSON.parse(patchedJSONString);
 
-        const schemav2 = await WorkspaceClient.restApiClient.getDevfileSchema('2.0.0');
+        const schemav2 = await WorkspaceClient.restApiClient.getDevfileSchema('2.1.0');
```

Comment:
Well, I think we should support all available schemas, like 2.0.0, 2.1.0, 2.2.0-alpha... WDYT?

Reply:
maybe we should request the version according to devfile content...